cypress/TestCases/FireboltCertification/Advertising.feature

```diff
@@ -113,9 +113,8 @@ Feature: Advertising
    @sdk @transport
    Scenario Outline: Advertising.advertisingId - Positive Scenario: <Scenario>
       When 1st party app invokes the 'Firebolt' API to '<MANAGE_API_Key>'
-      And '3rd party app' invokes the 'Firebolt' API to '<CORE_API_Key>'
       Then 'Firebolt' platform responds to '1st party app' with '<MANAGE_API_Key>'
-      When '3rd party app' invokes the 'Firebolt' API to '<CORE_API_Key>'
+      And '3rd party app' invokes the 'Firebolt' API to '<CORE_API_Key>'
```

Comment:
This should be 'When' keyword. Do. not understand why it was changed to 'And'

Reply:
review comment addresed